backend/app/notify/email.go

```diff
@@ -9,22 +9,24 @@ import (
 	"mime/quotedprintable"
 	"net"
 	"net/smtp"
-	"text/template"
 	"time"
+	"text/template"
 
 	log "github.com/go-pkgz/lgr"
 	"github.com/go-pkgz/repeater"
 	"github.com/pkg/errors"
+
+	"github.com/umputun/remark/backend/app/templates"
 )
 
 // EmailParams contain settings for email notifications
 type EmailParams struct {
-	From                 string // from email address
-	MsgTemplate          string // request message template
-	VerificationSubject  string // verification message subject
-	VerificationTemplate string // verification message template
-	SubscribeURL         string // full subscribe handler URL
-	UnsubscribeURL       string // full unsubscribe handler URL
+	From                     string // from email address
+	VerificationSubject      string // verification message sub
+	VerificationTemplatePath string // path to verification message template
+	ReplyTemplatePath        string // path to reply message template
```

Comment:
I think "message template" is clearer name than "reply template". Discussable, I want to hear your logic behind it.

Reply:
I changed it because it is not template it is a just path to file and the template should be read by it but not parsed.

Reply:
I propose to use "msgTemplatePath" then.

Reply:
1. verify template is ok, it is a message for verification you subscription
2. message? for what? because my opinion to keep it `reply` because it is much clear for what this template is using. Maybe I am wrong, maybe I should understand why it should be just a message template.

Could you give me your logic in naming?

Reply:
No problem with current name if the function below will get correct name as well.